### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by oliverSCZ**
I did the following: 

- Created a new JavaScript file for the new functionality.
- Implemented a function for adding a new task (add a new element to the array).
- Implemented a function for editing task descriptions.
- Implemented a function for deleting a task (remove an element from the array).
- Implemented a function for the "Clear all completed" button (use filter() method).
- Deleted a task updates all remaining items' indexes.
- All changes to the To Do List are saved in local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,46 +1,91 @@
...
+
+const taskIcon = Array.from(document.querySelectorAll('.dynamic-icons'));
+const trashIcons = [...document.querySelectorAll('.trash-icon')];
+const textDescription = [...document.querySelectorAll('.item-description')];
+const liItem = [...document.querySelectorAll('.item')];
+
+_.forEach(liItem, (item, i) => {
+  textDescription[i].addEventListener('focus', () => {
+    item.style.backgroundColor = 'rgba(248, 244, 2, 0.322)';
+    taskIcon[i].classList.add('display-icon');
```
Comment (EricMbouwe) [inline src/index.js:72]: - [ ] Editing task descriptions is not working as expected, the new task description is not persisted after the edit. Let's fix that

File: src/index.js [hunk 2]
```diff
@@ -1,46 +1,91 @@
...
-const getTaskFromLocal = () => JSON.parse(localStorage.getItem('task'));
+getTaskFromLocal();
 
-const populateList = (values) => {
-  values.forEach((toDo, i) => {
+const updateTasks = (data) => {
+  tasks = data;
+};
+
+export const removeAll = () => {
```
Comment (EricMbouwe) [inline src/index.js]: - [x] This function name doesn't reflect exactly its purpose (Remove all **completed** tasks) and rather bring some confusion. Let's give a meaningful name to this function to keep your code clean and simple.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
### Hi @oliverSCZ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight!
- Good PR ✔️
- Good commits ✔️

# Required Changes ♻️ ⛔️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (oliverSCZ) [reply src/index.js:72]: Is working, You only have to click on the task , write something or modify the taks and after that press TAB key or just click in other side, out of the task.

File: src/index.js (hunk 2 above)
Comment (oliverSCZ) [reply src/index.js]: Done

**Review by EricMbouwe — APPROVED** [review]
### Hi @oliverSCZ, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**